SpywareFilter/sections/specific.txt

```diff
@@ -39,6 +39,7 @@ tap.az##.links-sites > .counters
 ||tracking.pinflix.com/static/smt.js
 ||api.radio.*/info/*/user/log?event
 ||melonstube.com/js/analytics
+||wheredoyoucomefrom.ovh/private/counter.js
```

Comment:
Please move to Spanish filter, script contains functions for popunders.

Reply:
Definitely it a tracker. Block this domain completely.
![image](https://user-images.githubusercontent.com/8361299/60832011-97dc4280-a1c3-11e9-8de9-50f9546cfc8d.png)

Reply:
It must be included in both due to a) tracking and b) popup.